meta-balena-common/recipes-connectivity/balena-proxy-config/balena-proxy-config/redsocks-conf.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=redsocks.conf watcher service
+
+[Service]
+Type=oneshot
+ExecStart=/bin/echo 'redsocks.conf changed'
+ExecStart=/bin/systemctl restart redsocks-conf.target
+# PartOf cannot start/restart inactive units
+ExecStart=/bin/systemctl restart redsocks.service
```

Comment:
@markcorbinuk can you think of a more elegant solution to this? `PartOf` cannot start/restart inactive units and redsocks.service is always inactive if redsocks.conf doesn't exist.